build.zig

```diff
@@ -185,12 +185,14 @@ fn qemu_run_image_x86_64(b: *Builder, image_path: []const u8) *std.build.RunStep
         "--enable-kvm",
         "-drive", b.fmt("format=raw,file={}", .{image_path}),
         "-debugcon", "stdio",
+        "-vga", "virtio",
         //"-serial", "stdio",
-        "-m", "4G",
+        "-m", "1G",
+        "-display", "gtk,zoom-to-fit=off",
         "-no-reboot", "-machine",
         "q35",
         "-device", "qemu-xhci",
-        "-smp", "8",
+        "-smp", "1",
```

Comment:
Or this one?

Reply:
As for both, I used it for testing since I found QEMU to lag a lot on my system. Will solve in the next commit, thanks for noticing.